server.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const jsonParser = bodyParser.json();
```

Comment:
If not using the `jsonParser` it doesn't need to be defined. Some linters would flag this as an unused variable.